src/DotLiquid.Tests/ConditionTests.cs

```diff
@@ -266,10 +268,12 @@ public void TestCapitalInCustomCSharpOperator()
                 try
                 {
                     Condition.Operators["DivisibleBy"] =
-                        (left, right) => (int)left % (int)right == 0;
+                        (left, right) => (long)left % (long)right == 0;

```

Comment:
Keep the int test for backward compatibility and add the new test as a system convert to int64 to support both inputs.